patternfly/patternfly-elements
# Cpfed 3852 sticky nav test

---
name: Cpfed 3852 sticky nav test - add sticky nav feature and test top banner
labels: "feature"
---

## pfe-navigtation

- Added sticky nav feature and test RHEL Insights top banner. 
- Added sticky nav demo page. 
- Added content from redhat.com to the page so that it scrolled. 
- Updated overlay to fix issue were you could click on the content below the overlay when menu was open.

### Testing instructions

1. Pull down branch
2. Scroll up and down the page and you will see the sticky nav stick to the top and hide the banner when you scroll down the page then reshow the banner when you scroll back up.
3. Open menu then click on the overlay anywhere above content - should close menu but not activate any links or content on the page
4. Acceptance Criteria for feature was replicated from redhat.com

### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.
